packages/@lwc/engine-server/src/__tests__/fixtures.spec.ts

```diff
@@ -225,10 +225,6 @@ describe('fixtures', () => {
         testFixtures();
     }
 
-    describe('native custom element lifecycle', () => {
-        testWithFeatureFlagEnabled('ENABLE_NATIVE_CUSTOM_ELEMENT_LIFECYCLE');
-    });
-
```

Comment:
API versioning doesn't apply at all to SSR'd components in our current architecture, so there's little point in testing multiple API versions in SSR.